### PR TITLE
#2749 -  Add walkthrough for new dataset ui using introJS

### PR DESCRIPTION
**Pull request by marvinkome**
- [ ] **Update release notes if necessary**

Screenshots:
image

image

### CODE REVIEW

File: client/src/components/dataset/DatasetTableV2.jsx [hunk 1]
```diff
@@ -77,6 +80,61 @@ function DatasetTable(props) {
...
+      nextLabel: 'Next',
+      prevLabel: 'Back',
+      hidePrev: true,
+      skipLabel: 'Skip',
+      doneLabel: 'Got it',
+      showStepNumbers: false,
+      steps: [
+        {
+          element: '#GroupsList',
+          intro: 'Data groups are based on Flow form question groups and submission metadata',
```
Comment (valllllll2000) [inline client/src/components/dataset/DatasetTableV2.jsx:112]: Should the strings be in a strings resource file so we can have them translated?

File: client/src/components/dataset/DatasetTableV2.jsx [hunk 2]
```diff
@@ -77,6 +80,61 @@ function DatasetTable(props) {
...
+          dynamic: true,
+          position: 'right',
+        },
+        {
+          element: '.fixedDataTableLayout_main',
+          intro: "A selected data group's data os shown on the grid",
```
Comment (valllllll2000) [inline client/src/components/dataset/DatasetTableV2.jsx:118]: Should the strings be in a strings resource file so we can have them translated?

not sure if it's a typo: A selected data group's data **os** shown on the grid maybe should be  A selected data group's data **is** shown on the grid

Comment (marvinkome) [reply client/src/components/dataset/DatasetTableV2.jsx:118]: True

**Comment by marvinkome** [comment]
@valllllll2000 and @tangrammer can you add the French and Spanish translations also?

File: client/src/translations/en.json [hunk 3]
```diff
@@ -77,6 +77,8 @@
   "data_file": "Data File",
   "data_has_column_headers": "Data has column headers",
   "data_groups": "Data groups",
+  "data_group_intro_1": "Data groups are based on Flow form question groups and submission metadata. Lumen maintains all data relations and provides them when visualizing data.",
+  "data_group_intro_2": "A selected data group's data is shown on the grid",
```
Comment (tangrammer) [inline client/src/translations/en.json:81]: @marvinkome 
```
  "data_group_intro_1": "Los grupos de datos están basados en los grupos de preguntas de los formularios de Flow y en los envíos de meta información. Lumen mantiene todos las relaciones y las provee en las visualizaciones de datos"
  "data_group_intro_2": "Una selección de grupo de datos es mostrado en la cuadrícula",


```

@valllllll2000 do you agree with the Spanish version?

**Pull request merged**